src/Chapter2.hs

```diff
@@ -620,7 +644,11 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven l = go 0 l
+    where go _ [] = []
+          go 0 (x:xs) = x : go 1 xs
+          go 1 (_:xs) = go 0 xs
```

Comment:
Nice idea on the helper function :+1:

Alternatively, you could use pattern matching in `takeEven` instead. As you need to take every second element, it is possible to represent with a few pattern matching clauses 🙂